src/compiler/crystal/codegen/link.cr

```diff
@@ -112,7 +112,7 @@ module Crystal
     def self.default_rpath : String
       # do not call `CrystalPath.expand_paths`, as `$ORIGIN` inside this env
       # variable is always expanded at run time
-      ENV.fetch("CRYSTAL_LIBRARY_RPATH", "")
+      ENV.fetch("CRYSTAL_LIBRARY_RPATH", {{ env("CRYSTAL_CONFIG_LIBRARY_RPATH") || "" }})
```

Comment:
Why not add it into `Crystal::Config` as in the `library_path` case?

Reply:
I want to merge this with little friction. We can refactor it later.